pkg/cmd/extension/command.go

```diff
@@ -649,7 +659,7 @@ func checkValidExtension(rootCmd *cobra.Command, m extensions.ExtensionManager,
 
 	for _, ext := range m.List() {
 		if ext.Name() == commandName {
-			return ext, fmt.Errorf("there is already an installed extension that provides the %q command", commandName)
+			return ext, alreadyInstalledError
```

Comment:
Unfortunately this logic is not exactly correct. For example if two different users have extensions of the same name `samcoe/gh-triage` and `monalisa/gh-triage`, and we try to install one of them and then try to install the other one we will see the warning that the extension is already installed when in fact that is not true. 

We need to not only compare the name of the extension but also the owner of the extension. I don't think the extension owner is exposed in the extension interface yet but in order to get this to work I think we will need to make that change. Does that make sense?

Reply:
Yes, it does make sense. How do I expose the extension owner to the extension interface? I can try it.

Reply:
@Delta456 So I was in the middle of writing up instructions on how to add to the `extension` interface, but realized that becomes a lot more complicated than what I expected. Would you mind if I pushed a commit on top of yours implementing my idea?

Reply:
> @Delta456 So I was in the middle of writing up instructions on how to add to the `extension` interface, but realized that becomes a lot more complicated than what I expected. Would you mind if I pushed a commit on top of yours implementing my idea? 

Sure, go ahead!

Reply:
I fixed the CI for lint as well @samcoe!